scigym/envs/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# This can remain empty...
```

Comment:
Generally, it is recommended to add an empty line at the end of a file (see [here](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file))